data-serving/data-service/src/common/db-client.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb';
+
+class DbClient {
+    client: MongoClient;
+
+    constructor() {
+        this.client = new MongoClient('mongodb://localhost:27017', { useUnifiedTopology: true });
```

Comment:
is 27017 a hardcoded mongo thing?

Reply:
usually you'd pass the mongo url (which could also contain username and password) in an env variable.

Reply:
yep, hardcoding this here is temporary -- i'll use the dotenv package once ab merges it